docs/deployment/kubernetes.md

```diff
@@ -44,29 +46,30 @@ This step assumes you are running `kubectl` on a master host.
 
 * Clone the code
 
-```
-$ git clone https://github.com/openfaas/faas-netes
-```
+    ```bash
+    $ git clone https://github.com/openfaas/faas-netes
+    ```
 
-Deploy a stack with asynchronous functionality provided by NATS Streaming.
+    Deploy a stack with asynchronous functionality provided by NATS Streaming.
 
 * Deploy the whole stack
 
-This command is split into two parts so that the OpenFaaS namespaces are always created first:
+    This command is split into two parts so that the OpenFaaS namespaces are always created first:
 
-* openfaas - for OpenFaaS services
-* openfaas-fn - for functions
+    * openfaas - for OpenFaaS services
```

Comment:
What do these spaces indicate?

Reply:
I added 4 spaces to get the markdown to indent the configurations under each bullet point for readability. 
[Preview](https://deploy-preview-2--openfaas.netlify.com/deployment/kubernetes/#20b-deploy-openfaas)

Reply:
I added 4 spaces to get the markdown to indent the configurations under each bullet point for readability. 
[Preview](https://deploy-preview-2--openfaas.netlify.com/deployment/kubernetes/#20b-deploy-openfaas)